java_v1/App.java

```diff
@@ -0,0 +1,52 @@
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.HashSet;
+
+public class App {
+	
+	public static void main(String[] args) {
+
+		URL url = null;
+		HttpURLConnection conn = null;
+		HashSet<String> nodes = new HashSet<String>();
+		
+		String path = "https://www.raspberrypi.org/blog/the-little-computer-that-could/";
```

Comment:
Would it be possible to read this from an environmental variable and fallback to the URL as a default? It will help with the Docker container. 
i.e.
```
    String urlOverride= System.getenv("targetUrl");
```